README.md

```diff
@@ -26,104 +26,47 @@ Examples:
+On successful completion of the Docker task, the output file will be
 automatically de-localized from:
 
-    /mnt/data/output/gs
-
-In the above examples, the output files would be de-localized from:
-
-    /mnt/data/output/gs/mybucket/mypath/
-
-Support for wildcards *on filenames* is available for both inputs and outputs.
-For example:
-
-    --input gs://mybucket/mypath/*.bam
-    --output gs://mybucket/mypath/*.bam.bai
-
-#### Recursive copy
-
-Support for recursive copy is available for inputs and outputs.
-
-For inputs, this allows you to pull an entire tree of inputs from
-Google Cloud Storage (GCS), for example:
+    /mnt/data/output/gs/mybucket/mypath/sample1.stats.txt
 
-    dsub ... \
-      --input-recursive INPUT_PATH=gs://bucket/path
+More support is available for input and output handling such as:
 
-Your pipelines script will get a variable INPUT_PATH, which contains the
-on-disk location, `/mnt/data/input/gs/bucket/path`.
+  * wildcards on filenames
+  * recursive directory copying
 
-For outputs, this allows you to generate output with subdirectories and
-dsub will recursively copy your output to GCS. For example:
+See the [Input/Output docs](docs/input_output.md) for more details.
 
-    dsub ... \
-      --output-recursive OUTPUT_PATH=gs://bucket/path
+#### Script variables
 
-Your pipelines script will get a variable OUTPUT_PATH, which contains the
-on-disk location, `/mnt/data/output/gs/bucket/path`.
+In the above example, your Docker script will receive environment variables
+`INPUT_FILE` and `OUTPUT_FILE` set automatically.
 
-As a convenience, if a recursive input or output parameter is passed, dsub will
-automatically install the [Google Cloud SDK](https://cloud.google.com/sdk/docs/)
-at runtime (before your script executes). For large jobs, you are encouraged
-to install gcloud in your Docker image when it is built.
-
-#### Unsupported path formats:
-
-* GCS recursive wildcards (**) are not supported
-* Wildcards in the middle of a path are not supported
-* Output parameters to a directory are not supported, instead:
-  * use an explicit wildcard on the filename (such as `gs://mybucket/mypath/*`)
-  * use the recursive copy feature
-
-#### Script parameters
-
-If your Docker task script needs to refer to inputs or outputs explicitly,
-names can be provided for the input and output parameters. These
-parameters will be available as environment variables in the Docker container.
-
-For example:
-
-    --input OLD_VCF=gs://mybucket/mypath1/sample.vcf \
-    --input NEW_VCF=gs://mybucket/mypath2/sample.vcf \
-    --output OUTPUT=gs://mybucket/mypath/*
-
-will result in the following environment variables being set:
-
-    OLD_VCF=/mnt/data/input/gs/mypath1/sample.vcf
-    NEW_VCF=/mnt/data/input/gs/mypath2/sample.vcf
-    OUTPUT=/mnt/data/output/gs/mypath/*
-
-To pass simple environment variables to your jobs, use the `--env` parameter:
+To pass simple, non-file, values to your jobs, use the `--env` parameter:
```

Comment:
You can reference the `--input`, `--output`, and `--env` variables from your script using standard bash notation, like `${INPUT_FILE}` and `${OUTPUT_FILE}`.